runtime/runtime-params-estimator/src/cases.rs

```diff
@@ -156,6 +156,41 @@ pub fn run(mut config: Config, only_compile: bool) -> RuntimeConfig {
             contract_byte_cost,
             ratio_to_gas(config.metric, contract_compile_base_cost)
         );
+        process::exit(0);
+    } else if only_evm {
+        let cost = cost_of_evm(config.metric, true);
+        println!(
+            "EVM base deploy (and init evm instance) cost: {}, deploy cost per EVM gas: {}",
+            ratio_to_gas(config.metric, cost.deploy_cost.1),
+            ratio_to_gas(config.metric, cost.deploy_cost.0)
```

Comment:
Lets update `Coef` to be a struct with `mean` and `base` named params

Reply:
Good idea. Same for near part gas estimation? which uses a unaliased `(Ratio<u64>, Ratio<u64>)`